src/capture.js

```diff
@@ -30,6 +30,12 @@ function listen (appRoot, port, options) {
             return !!s;
         }).join('/');
 
+        // If the final argument ended with a trailing slash, return it.
+        var last = args.slice(-1)[0];
+        if (last.slice(-1) === '/') {
```

Comment:
So the reason for the failing test is related to when the last argument is `/`.

Updating the condition to read as below does the trick:

```js
if (last !== '/' && last.slice(-1) === '/') {
    path += '/';
}
```

Do you mind rebasing off `master` and applying the suggested change?